internal/daemon/worker/session/session.go

```diff
@@ -107,6 +107,7 @@ type sess struct {
 	resp        *pbs.LookupSessionResponse
 	status      pbs.SESSIONSTATUS
 	cert        *x509.Certificate
+	sessionId   string
```

Comment:
Curious why this is needed? I don't see a problem with having it stored here but not sure what this is fixing given we have it from `resp` already.

Reply:
The tests panic'ed with a lock on GetId; since that shouldn't change I figure we can set it once when we add the session to the map and don't need a lock as that won't change. All the other Gets lock and access the resp

Reply:
Hm -- any chance you can tell me how to repro that? I'm curious as to why they panic'd.